epytope/EpitopePrediction/External.py

```diff
@@ -1454,9 +1457,9 @@ def parse_external_result(self, file):
         f = csv.reader(open(file, "r"), delimiter = '\t')
         scores = defaultdict(defaultdict)
         ranks = defaultdict(defaultdict)
-        alleles = [Allele(x[:5]+'*'+x[5:]) for x in next(f) if x != ""]
+        alleles = [x for x in next(f) if x != ""]
```

Comment:
I think it would be good if you would add information about the result dictionary to the `parse_external_result` method signature. Just to make sure that it is clear that we have `allele strings` etc.